pkg/geo/OWNERS

```diff
@@ -0,0 +1,2 @@
+owners:
+  - geospatial
```

Comment:
ditto here. Is there some `yaml` restriction that we can't have this be just

```
geospatial
```

I think that should work as this is a valid yaml document (representing `"geospatial"`).

With two teams,

```
- geospatial
- lololo
```

I dislike that this can't be

```
geospatial
lololo
```

as this is what a non-yaml'ers intuition would dictate.

Reply:
That's very complicated to parse, especially with multiple file owners :\ (we unfortuantely can't get away from this because the sql package will require fine grained permissions)